src/mscorlib/shared/System/Span.NonGeneric.cs

```diff
@@ -22,6 +23,90 @@ namespace System
     /// </summary>
     public static class Span
     {
+        /// <summary>
+        /// Determines whether the beginning of the span matches the specified value when compared using the specified comparison option.
+        /// </summary>
+        public static bool StartsWith(this ReadOnlySpan<char> span, ReadOnlySpan<char> value, StringComparison comparison)
```

Comment:
Some of the other extensions methods in this file (e.g. `AsBytes`) uses `source` instead of `span` for the `this` parameter name. It'd be nice if the naming was consistent.

Reply:
The APIs where `this` is `source` have another span parameter as `destination`. The other ones don't. That is the reason for the inconsistency.  If we feel strongly about the parameter renaming, I can do it in all methods.

cc @KrzysztofCwalina, @terrajobst